### DISCUSSION TITLE
How to add custom css to one page in papermod?

### DISCUSSION THREAD

**Discussion by MaxCCC** [adityatelange/hugo-PaperMod#470] [Q&A]
I have used this hugo template: https://github.com/adityatelange/hugo-PaperMod. Now I want to add some CSS to my post.md.

First I needed HTML. I added layouts/rawhtml.html content:

```
<!-- raw html -->
{{.Inner}}
```

The HTML works in my post.md:

```
{{< rawhtml >}}
<html>
<head>
  <link rel="stylesheet" href="css/test.css">
</head>
<body>

<h1>This is a heading</h1>
<p>This is a paragraph.</p>

</body>
</html> 
{{< /rawhtml >}}
```

But how can I import my test.css? I've created a folder with my css file assets/css/test.css. I tried multiple paths but I can't get it working.

**Comment by pixelfabs** [comment]
I don't think what you do is even possible, since you add a completely new HTML page in another HTML page. 

rawhtml is for adding anything that normally comes in between `<body></body>`. Start & end of HTML file is already in baseof.html

If you need to style a part of your page, do this instead:

```
{{< rawhtml >}}
<div class="special-markup">
Lorem Ipsum is simply dummy text of the printing and typesetting industry.
</div>
{{< /rawhtml >}} 
```

Then add `.special-markup`  to assets/css/extended/test.css

If you need a page that completely looks different, do this instead:

make a `layout/_default/special-page.html`

then add `layout: special-page` to `special-post.md`
